.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discussions
+    url: https://github.com/streetcomplete/StreetComplete/discussions
+    about: For discussions about the usage of StreetComplete, asking questions or talking about visions.
+  - name: Translations
+    url: https://poeditor.com/join/project/IE4GC127Ki
```

Maybe better link to the relevant section in the contributing guide instead?

```suggestion
    url: https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app```

---

Fine by me. But then CONTRIBUTING.md should also say that translation discussions should be done on POEditor, no?

Side note: could you please edit your suggestion to remove the surplus 3 backticks (```) at the end.

---

I checked it and it doesn't say anything about discussing translations.

---

Feel free to modify/update the text there, too!

@riQQ You can use this link to keep the change in this PR: https://github.com/riQQ/StreetComplete/edit/add-issue-template-config/CONTRIBUTING.md